[2023-11-15T04:25:51.483+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: recalls_etl_s3.clean_raw_data manual__2023-11-15T04:25:41.907414+00:00 [queued]>
[2023-11-15T04:25:51.493+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: recalls_etl_s3.clean_raw_data manual__2023-11-15T04:25:41.907414+00:00 [queued]>
[2023-11-15T04:25:51.493+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-15T04:25:51.510+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): clean_raw_data> on 2023-11-15 04:25:41.907414+00:00
[2023-11-15T04:25:51.517+0000] {standard_task_runner.py:57} INFO - Started process 473 to run task
[2023-11-15T04:25:51.520+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'recalls_etl_s3', 'clean_raw_data', 'manual__2023-11-15T04:25:41.907414+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/recalls_etl_s3.py', '--cfg-path', '/tmp/tmpe3wkej4a']
[2023-11-15T04:25:51.522+0000] {standard_task_runner.py:85} INFO - Job 56: Subtask clean_raw_data
[2023-11-15T04:25:51.577+0000] {task_command.py:416} INFO - Running <TaskInstance: recalls_etl_s3.clean_raw_data manual__2023-11-15T04:25:41.907414+00:00 [running]> on host c912e3390c69
[2023-11-15T04:25:51.672+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='recalls_etl_s3' AIRFLOW_CTX_TASK_ID='clean_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-15T04:25:41.907414+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-15T04:25:41.907414+00:00'
[2023-11-15T04:25:53.474+0000] {logging_mixin.py:151} INFO - Success: cleaned 29270 recall records.
[2023-11-15T04:25:53.480+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-15T04:25:53.495+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=recalls_etl_s3, task_id=clean_raw_data, execution_date=20231115T042541, start_date=20231115T042551, end_date=20231115T042553
[2023-11-15T04:25:53.541+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-15T04:25:53.566+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
